src/Feature/Changelog/Feature_Changelog.re

```diff
@@ -0,0 +1,397 @@
+    | Text(_) => failwith("Unexpected text node");
+
+  let parse =
+    fun
+    | Element("changelog", _, children) => List.map(parseCommit, children)
+    | Element(_) => failwith("Unexpected element")
+    | Text(_) => failwith("Unexpected text node");
+
+  let path = Revery.Environment.getAssetPath("changelog.xml");
+  let simpleXml =
+    Markup.file(path) |> fst |> Markup.parse_xml |> Markup.signals |> simplify;
+
+  switch (parse(simpleXml)) {
+  | commits => Ok(commits)
+  | exception (Failure(message)) => Error(message)
+  };
+};
+
+// VIEW
+
+let model = Lazy.from_fun(read);
+
+module View = {
+  open Revery.UI;
+  open Revery.UI.Components;
+  open Oni_Components;
+
+  module Colors = Feature_Theme.Colors;
+
+  // STYLES
+
+  module Styles = {
+    open Style;
+
+    let scrollContainer = [flexGrow(1)];
+
+    let content = [
+      padding(10),
+      paddingLeft(20),
+      paddingRight(20),
+      overflow(`Hidden),
+    ];
+
+    let commit = [flexDirection(`Row), marginTop(10)];
+
+    let groupHeader = (font: UiFont.t, ~theme) => [
+      fontFamily(font.fontFile),
+      fontSize(16.),
+      color(Colors.foreground.from(theme)),
+      marginTop(16),
+    ];
+
+    let groupBody = [paddingLeft(10)];
+
+    let typ = (font: UiFont.t, ~color) => [
+      fontFamily(font.fontFile),
+      fontSize(12.),
+      Style.color(color),
+      width(60),
+    ];
+
+    let scope = (font: UiFont.t, ~theme) => [
+      fontFamily(font.fontFile),
+      fontSize(12.),
+      width(130),
+      color(
+        Colors.foreground.from(theme) |> Revery.Color.multiplyAlpha(0.75),
+      ),
+    ];
+
+    let summary = (font: UiFont.t, ~theme) => [
+      fontFamily(font.fontFile),
+      fontSize(12.),
+      color(Colors.foreground.from(theme)),
+    ];
+
+    let breakingCommit = [marginTop(10), marginBottom(6)];
+
+    let breaking = [flexDirection(`Row), marginTop(4)];
+
+    let breakingText = (font: UiFont.t, ~theme) => [
+      fontFamily(font.fontFile),
+      fontSize(12.),
+      color(Colors.foreground.from(theme)),
+      marginLeft(10),
+    ];
+
+    let error = (font: UiFont.t) => [fontFamily(font.fontFile)];
+  };
+
+  let date = (~commit, ~style, ()) => {
+    let time = Unix.localtime(commit.time);
+    let text =
+      Printf.sprintf(
+        "%u-%02u-%02u",
+        time.tm_year + 1900,
+        time.tm_mon + 1,
+        time.tm_mday,
+      );
+    <Text style text />;
+  };
+
+  let typ = (~commit, ~uiFont, ~theme, ()) => {
+    let text =
+      switch (commit.typ) {
+      | Some("feat") => "feature"
+      | Some("fix") => "bugfix"
+      | Some(other) => other
+      | None => "unknown"
+      };
+
+    let color =
+      switch (commit.typ) {
+      | Some("feat") => Colors.Oni.visualModeBackground.from(theme)
+      | Some("fix") => Colors.Oni.operatorModeBackground.from(theme)
+      | Some("perf") => Colors.Oni.normalModeBackground.from(theme)
+      | _ => Colors.foreground.from(theme)
+      };
```

Comment:
I like the detail and thought you put into the formatting - the screenshots look great!